### ISSUE TITLE
When I create a new measure in DAX Studio, I can't change the name.  Now my calculated measures are named "MyMeasure", "MyMeasure1", etc.

### ISSUE THREAD

**Issue by danielpitz** [DaxStudio/DaxStudio#1282]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by dgosbell** [comment]
Thanks for logging this. I'm assuming you are referring to measures created in the Query Builder since those are the ones that get "MyMeasure", "MyMeasure1", etc as the generated names.

**Comment by danielpitz** [comment]
Yes, it refers to measures created in the Query Builder.  I thought maybe it had something to do with the 3.1 versions, so I re-installed 3.0.11, but the behavior was still the same.

**Comment by danielpitz** [comment]
Did I need to provide more details?  I noticed this is closed, but I don't know what the solution is.
Thanks.

**Comment by dgosbell** [comment]
This is fixed in the v3.2 release which I just pushed out. When I tag an issue in the release it gets automatically closed by github
